src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -194,6 +201,10 @@ public function denormalize($data, $class, $format = null, array $context = arra
             }
         }
 
+        if (!empty($extraAttributes)) {
+            throw new ExtraAttributesException(sprintf('Extra attributes are not allowed ("%s" are unknown).', implode('", "', $extraAttributes)));
```

Comment:
Given you have specific exception type, it makes sense to put such logic into the exception, i.e. `throw new ExtraAttributesException($extraAttributes)` (see `Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException` for example).